packages/admin/blocks-admin/src/blocks/factories/columnsBlock/FinalFormLayoutDisplay.tsx

```diff
@@ -0,0 +1,35 @@
+import { ListItemText } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import React from "react";
+import { FieldRenderProps } from "react-final-form";
+
+import { ColumnsBlockLayout } from "../createColumnsBlock";
+
+interface Props extends FieldRenderProps<ColumnsBlockLayout> {
+    layouts: ColumnsBlockLayout[];
+}
+
+export function FinalFormLayoutDisplay({ layouts }: Props): React.ReactElement {
+    return (
+        <LayoutDisplayContainer>
+            {layouts[0].preview}
+            <StyledListItemText primary={layouts[0].label} secondary={layouts[0].name} />
+        </LayoutDisplayContainer>
+    );
+}
+
+const LayoutDisplayContainer = styled("div")`
+    display: grid;
+    grid-template-columns: minmax(80px, 1fr) 2fr;
+    column-gap: ${({ theme }) => theme.spacing(2)};
+    align-items: center;
+    background-color: #ffffff;
+    border: 1px solid #d9d9d9;
```

Comment:
You can use the `divider` color from the theme here:

```suggestion
    border: 1px solid ${({ theme }) => theme.palette.divider};
```